cypress/page/student-registration-form.page.ts

```diff
@@ -0,0 +1,83 @@
+import { data } from "cypress/types/jquery";
+import {PersonalInformation} from "../interfaces/PersonalInformation";
+class StudentRegistrationForm {
+  private nameText: string;
+  private lastNameText: string;
+  private emailText: string;
+  private genderBtn: string;
+  private mobileNumberText: string;
+  private hobbiesCheck: string;
+  private currentAddressText: string;
+  private submitBtn: string;
+  private stateList: string;
+  private cityList: string;
+  private NCROstate: string;
+  private delhiCity: string;
+  private hobbies: string;
+  private myhashhobbies: Map;
+
+  constructor() {
+    this.studentFormPageURL = 'https://demoqa.com/automation-practice-form';
+    this.bodyTr = 'tbody tr';
+    this.nameText = '#firstName'
+    this.lastNameText = '#lastName';
+    this.emailText = '#userEmail';
+    this.genderBtn = '[type="radio"]';
+    this.mobileNumberText = '#userNumber';
+    this.hobbiesCheck = '[type="checkbox"]';
```

Comment:
if you change the selector for hobbies by `#hobbiesWrapper input+label` and use a contains, you should not worry about by the value number corresponding to a hobbie

Reply:
resolved